primitives/src/market.rs

```diff
@@ -28,7 +28,7 @@ use sp_runtime::RuntimeDebug;
 /// * `BN`: Block Number
 /// * `M`: Moment (Time moment)
 #[derive(Clone, Decode, Encode, Eq, PartialEq, RuntimeDebug, TypeInfo)]
-pub struct Market<AI, BN, M> {
+pub struct Market<AI, Balance, BN, M> {
```

Comment:
Not quite sure how to handle this. I'm surprised to find that https://rust-lang.github.io/api-guidelines/naming.html suggests "concise `UpperCamelCase`, usually single uppercase letter: `T`" for type parameters. Style is inconsistent even in `markets.rs`. For example, `MarketDispute` uses `UpperCamelCase`, other use abbreviations. What do you think it should be?

I'm leaning towards `UpperCamelCase`, because the only reason we even need the documentation is because we're using abbreviation right now: If `AI` was `AccountId`, there'd be no need for documentation like "AI is the AccountId".

Reply:
Both ways have their pros and cons. Abbrevs are smaller especially useful when you have a lot of Generics. On the other hand full names are much clearer to understand and don't need this documentation mentioned. What we should do is stay consistent, so either we change it all or we use abbrevs.

Reply:
Let's stick with (documented) abbreviations for now. This is what I see being used in most projects. As a rule of thumb we can apply as many letters as necessary to differentiate between the generics. For example, the minimum abbreviation here would be `A, BA, BN, M`. An alternative would be to use the longest abbreviation length as a default `AI, BA, BN, MA` to keep lengths consistent,

Reply:
[b6807d9](https://github.com/zeitgeistpm/zeitgeist/pull/892/commits/b6807d9bfcaeae07bb4a1ddbf05f39eafc5361ec)